src/components/layouts/collection-page-layout.tsx

```diff
@@ -257,8 +256,13 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
     [],
   ).map((lesson: LessonResource) => lesson.slug)
 
-  const {name, avatar_url, url: instructor_slug, bio_short, twitter} =
-    instructor || {}
+  const {
+    full_name: name,
+    avatar_url,
+    slug,
```

Comment:
converted `name` to `full_name` in the sanity query to match the rails `full_name` attribute that comes in.

Reply:
similarly, using `slug` to be consistent with rails